source/extensions/filters/http/dynamo/dynamo_filter.cc

```diff
@@ -176,11 +176,13 @@ void DynamoFilter::chargeStatsPerEntity(const std::string& entity, const std::st
       time_source_.monotonicTime() - start_decode_);
 
   size_t group_index = DynamoStats::groupIndex(status);
+  Stats::StatNameDynamicPool dynamic(stats_->symbolTable());
```

Comment:
For my own understanding, in the *not contended* case, what is the perf implication of this change? Is it more costly now to have to recompute a bunch of stuff on each request? Could you potentially cover some of these tradeoffs in stats.md?